tests/test_models.py

```diff
@@ -19,8 +19,31 @@
 from enterprise.models import (EnterpriseCustomer, EnterpriseCustomerBrandingConfiguration, EnterpriseCustomerUser,
                                PendingEnterpriseCustomerUser, logo_path)
 from test_utils.factories import (EnterpriseCustomerFactory, EnterpriseCustomerIdentityProviderFactory,
-                                  EnterpriseCustomerUserFactory, PendingEnterpriseCustomerUserFactory,
-                                  UserDataSharingConsentAuditFactory, UserFactory)
+                                  EnterpriseCustomerUserFactory, PendingEnrollmentFactory,
+                                  PendingEnterpriseCustomerUserFactory, UserDataSharingConsentAuditFactory,
+                                  UserFactory)
+
+
+@mark.django_db
+@ddt.ddt
+class TestPendingEnrollment(unittest.TestCase):
+    """
+    Test for pending enrollment
+    """
+    @ddt.data(
+        str, repr
+    )
+    def test_string_conversion(self, method):
```

Comment:
@haikuginger looks like this test does not actually test the only "sensible" method `coplete_enrollment`. Please find some time to add it.